packages/core/stories/patterns/selectable-card/selectable-card.stories.tsx

```diff
@@ -0,0 +1,124 @@
+import {
+  H3,
+  Text,
+  StackLayout,
+  InteractableCard,
+  InteractableCardGroup,
+  InteractableCardValue,
+  Avatar,
+  CheckboxIcon,
+  RadioButtonIcon,
+} from "@salt-ds/core";
+import { CreditCardIcon, BankIcon } from "@salt-ds/icons";
+import { Meta, StoryFn } from "@storybook/react";
+import { useState } from "react";
+
+import persona1 from "../../assets/avatar.png";
+import persona2 from "../../assets/avatar2.png";
+import persona3 from "../../assets/avatar3.png";
+
+export default {
+  title: "Patterns/Selectable Card",
+} as Meta;
+
+export const SingleSelection: StoryFn<typeof InteractableCard> = (args) => {
+  const [selected, setSelected] = useState<InteractableCardValue>();
+
+  return (
+    <StackLayout>
+      <StackLayout gap={1}>
+        <H3 style={{ margin: 0 }}>Payment method</H3>
```

Comment:
We should either add the text to multiselect or remove it from here

Reply:
updated